compiler.py

```diff
@@ -102,9 +103,28 @@ def proc_file(file, files, mod_name, injects=None):
 
 
 def apply_merge(a, b):
-    content = a['content']
+    #Find variable definitions in b (file to be merged)
+    bVars=""
+    bRest=""
+    for line in b['content'].split("\n"):
+        if line.startswith("var "):
+            bVars+=line+"\n"
+        else:
+            bRest+=line+"\n"
+
+    if bVars!="":
+        firstVar = a['content'].find("var")
```

Comment:
find("var") could be kinda risky, maybe needs a regex instead like `^\s*var\(|\s`